### ISSUE TITLE
internalError unconsistent stack size 

### ISSUE THREAD

**Issue by XiaoYeZiZhang** [bellard/quickjs#119]
Hi there,  qjs throws an error InternalError: unconsistent stack size: 2 3 (pc=45) on the following case:
```
function* test() {
	try {
		if(0 == (yield 1)) return;
	} finally {
		return;
	}
}
test();
```

but it seems ok if the case is:
```
function* test() {
	try {
		if((yield 1) == 0) return;
	} finally {
		return;
	}
}
test();
```

How can I fix this unconsistent stack size error? 

Best regards and thanks for the great work.

**Comment by bellard** [comment]
fixed (see issue #166)

**Comment by XiaoYeZiZhang** [comment]
您好，我是张也，我已经收到您的邮件，我会及时查收并回复，谢谢~~
